watchdog/afterburn_request_handler.go

```diff
@@ -35,6 +36,16 @@ func makeAfterburnRequestHandler(config *WatchdogConfig) func(http.ResponseWrite
 		log.Fatalln(readPipeErr)
 	}
 
+	errPiper, errPipeErr := process.StderrPipe()
+	if errPipeErr != nil {
+		log.Fatalln(errPipeErr)
+	}
+
+	// stderr := NewCapturingPassThroughWriter(os.Stderr)
+	go func() {
+		io.Copy(os.Stdout, errPiper)
+	}()
+
```

Comment:
Not heard from @genexp in a while. @rgee0 can we log a suggestion to capture "stdout" from the watchdog in "afterburn mode" please?